R/pkg/R/functions.R

```diff
@@ -2095,26 +2061,28 @@ setMethod("atan2", signature(y = "Column"),
             column(jc)
           })
 
-#' datediff
+#' @section Details:
+#' \code{datediff}: Returns the number of days from \code{start} to \code{end}.
 #'
-#' Returns the number of days from \code{start} to \code{end}.
-#'
-#' @param x start Column to use.
-#' @param y end Column to use.
-#'
-#' @rdname datediff
-#' @name datediff
-#' @aliases datediff,Column-method
-#' @family date time functions
+#' @rdname column_datetime_functions
+#' @aliases datediff datediff,Column-method
 #' @export
-#' @examples \dontrun{datediff(df$c, x)}
+#' @examples
+#'
+#' \dontrun{
+#' set.seed(11)
+#' tmp <- createDataFrame(data.frame(time_string1 = as.POSIXct(dts),
+#'              time_string2 = as.POSIXct(dts[order(runif(length(dts)))])))
+#' tmp2 <- mutate(tmp, timediff = datediff(tmp$time_string1, tmp$time_string2),
+#'                monthdiff = months_between(tmp$time_string1, tmp$time_string2))
+#' head(tmp2)}
 #' @note datediff since 1.5.0
-setMethod("datediff", signature(y = "Column"),
-          function(y, x) {
-            if (class(x) == "Column") {
-              x <- x@jc
+setMethod("datediff", signature(x = "Column"),
+          function(x, y) {
```

Comment:
Here, `x` and `y` are reversed for easy documentation. Similarly for other methods that take two arguments.